common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -1,241 +1,71 @@
 (function(define) {
     'use strict';
-// VideoAccessibleMenu module.
+// VideoTranscriptDownloadHandler module.
     define(
-'video/035_video_accessible_menu.js', [],
-function() {
+'video/035_video_accessible_menu.js', ['underscore'],
+function(_) {
     /**
      * Video Download Transcript control module.
      * @exports video/035_video_accessible_menu.js
      * @constructor
      * @param {jquery Element} element
      * @param {Object} options
      */
-    var VideoAccessibleMenu = function(element, options) {
-        if (!(this instanceof VideoAccessibleMenu)) {
-            return new VideoAccessibleMenu(element, options);
+    var VideoTranscriptDownloadHandler = function(element, options) {
+        if (!(this instanceof VideoTranscriptDownloadHandler)) {
+            return new VideoTranscriptDownloadHandler(element, options);
         }
 
-        _.bindAll(this, 'openMenu', 'openMenuHandler', 'closeMenu', 'closeMenuHandler', 'toggleMenuHandler',
-            'clickHandler', 'keyDownHandler', 'render', 'menuItemsLinksFocused', 'changeFileType', 'setValue'
-        );
+        _.bindAll(this, 'bindHandlers', 'clickHandler');
```

Comment:
Let's get rid of "bindHandlers" from here and just inline the single line of code.